src/bot.py

```diff
@@ -45,7 +49,8 @@ def show_top_avatar(self, data) -> None:
 
         # Retrieving the user's avatar and saving it.
         top_contributor_avatar = data[0]['avatar_url']
-        urllib.request.urlretrieve(top_contributor_avatar, 'top_contributor_avatar.png')
+        avatar = top_contributor_avatar
```

Comment:
How about only using one variable in this case?

Reply:
ok